athena-neptune/src/main/java/com/amazonaws/athena/connectors/neptune/TypeRowWriter.java

```diff
@@ -38,18 +38,19 @@
 
 import java.util.ArrayList;
 import java.util.Map;
+
 /**
  * This class is a Utility class to create Extractors for each field type as per
  * Schema
  */
-public final class TypeRowWriter
+public final class TypeRowWriter 
```

Comment:
i think your IDE is using a different style than we are. can you remove these extra whitespace changes? there are many of them

Reply:
Will update the code by removing white spaces.